extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -2107,11 +2107,11 @@ function() use ( $dbr, $condition, $fname ) {
 		}
 
 		if ( !empty( $city_id ) ) {
-			$oRow2 = WikiaDataAccess::cache(
+			$oRow = WikiaDataAccess::cache(
 				static::getVarValueKey( $city_id, $oRow->cv_id ),
 				WikiaResponse::CACHE_STANDARD,
 				function() use ($dbr, $oRow, $city_id, $fname) {
-					return $dbr->selectRow(
+					$row = $dbr->selectRow(
```

Comment:
we should return this value

Reply:
It is returned - https://github.com/Wikia/app/pull/15277/files#diff-d12a757d67d597263b191479f98ecc01R2136 :)